install-ci-tools.sh

```diff
@@ -93,7 +93,7 @@ EOF
       if [ ! -x yarn ] || [ "$(./yarn -V)" != "$version" ]; then
         rm -rf yarn .yarn
         mkdir -p .yarn
-        curl -sSL "https://github.com/yarnpkg/yarn/releases/download/v${version}/yarn-v${version}.tar.gz" | tar xz -C .yarn --strip 1
+        retry curl -sSL "https://github.com/yarnpkg/yarn/releases/download/v${version}/yarn-v${version}.tar.gz" | tar xz -C .yarn --strip 1
```

Comment:
`retry` won't exist in this context, so use curl's `--retry` as you mentioned please

Reply:
🤔  because it's [defined below](https://github.com/goodeggs/travis-utils/blob/6e3318d54b5859e703749682bfc1dd5993e92d4b/install-ci-tools.sh#L121-L122)? Couldn't I just move the declaration up? What do you think about the relative merits of the two retry strategies?